mod-mosaic/schemas/mosaic_order_request.json

```diff
@@ -0,0 +1,150 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Mosaic Configuration",
+  "type": "object",
+  "javaName": "MosaicConfiguration",
+  "properties": {
+    "id": {
+      "description": "UUID of the configuration",
```

Comment:
Description should be updated to UUID of order

Reply:
thanks, updated